validator/src/main.rs

```diff
@@ -1235,6 +1237,13 @@ pub fn main() {
                 .validator(is_parsable::<usize>)
                 .help("Controls the TPU connection pool size per remote addresss"),
         )
+        .arg(
+            Arg::with_name("staked_nodes_overrides")
+                .long("staked-nodes-overrides")
+                .value_name("FILE_OR_URL")
+                .takes_value(true)
+                .help("Provide file with custom overrides for stakes of specific IPs."),  // TODO: change IP to IP or identity
```

Comment:
It'd be nice to include some more info in the `.help()` about what impact this overrides may have for the validator operationally, either as an inline or a call out to external documentation.

Reply:
I added the more instructive description. I will be glad if you can comment on the way I did.

I searched for the documentation page that the description could be elaborated (not sure if it's the way to go now) but I haven't found any CLI reference guide where I could add that to. The solana cli has got the reference page that seems to be generated from the code (https://docs.solana.com/cli/usage#solana-cli). The `solana-validator` has got no reference guide. Should that added to some specific guide with even more details?